_faq/student-obligations.md

```diff
@@ -3,10 +3,9 @@ question: "What are my obligations as a student during the GSoC?"
 ---
 We expect you to:
 
-* Be active on [gitter](https://coala.io/chat).
-* Blog at least once per week (this will be aggregated on
-[blog.coala.io](https://blog.coala.io)).
-* Participate in weekly meetings with your mentor.
+* Be active on matrix, specifically in [#gsoc:matrix.org](https://matrix.to/#/!WZbLERcNJxrvkxPfyV:matrix.org?via=sw1v.org&via=matrix.org&via=thebeckmeyers.xyz).
+* Blog at least once per week.
```

Comment:
@benparsons I have added this as it's what many other orgs use, do you want to encourage this and link to a blog each week from TWIM?